### DISCUSSION TITLE
Blackbox logs not readable

### DISCUSSION THREAD

**Discussion by theproff777** [iNavFlight/inav#6927] [Q&A]
[LOG00090.TXT](https://github.com/iNavFlight/inav/files/6349655/LOG00090.TXT)

There seems to be an issue, either with how the blackbox logs are recorded or with the blackbox log viewer not being able to read the logs.

I have attached a log file that I am not able to 'read' in the explorer.

---
# version
# INAV/OMNIBUSF7V2 2.6.0 Dec  4 2020 / 12:49:15 (cada68e4)
# GCC-9.2.1 20191025 (release) [ARM/arm-9-branch revision 277599]

- FC Board name and vendor: Omnibus F4 Pro V3
- INAV version string: INAV 2.6.0 (cada68e4) OMNIBUSF7V2

- Blackbox Explorer 3.4.0

**Comment by DzikuVx** [comment]
Either logging rate is too fast of SD card is too slow. Lower logging rate and/or replace SD card. SanDisk Ultra 16GB are proven to work fine

**Reply by theproff777** [reply]
I am using a SanDisk Ultra 16GB Class 10 as per your recommendation I saw some time ago. Formatted with SD Card Formatter.

Interestingly, the configurator will not let me set anything more than 25%, and so that was recorded at 25%.

I am waiting for a replacement motor to arrive and then I will try again at the lowest rate possible to see if doing that works.

**Reply by theproff777** [reply]
Finally got the motor installed and a flight in and this time the logging worked.

I did not change anything but the motor, same SD card and logging rate of 25% as before.

Really weird. Will be flying again tomorrow and will check again to see it is still working.

**Reply by theproff777** [reply]
Flown a few times now and all working as expected.
